receiver/vcenterreceiver/metadata.yaml

```
@@ -382,3 +382,19 @@ metrics:
       aggregation: cumulative
     attributes: []
     extended_documentation: As measured over the most recent 20s interval.
+  vcenter.vm.cpu.utilization:
+    enabled: true
+    description: The CPU utilization of the VM.
+    unit: "%"
+    gauge:
+      value_type: double
+    attributes: []
+  vcenter.vm.cpu.usage:
+    enabled: true
+    description: The amount of CPU in Hz used by the VM.
```

```suggestion
    description: The amount of CPU used by the VM.
```
Generally, we don't include the units in the description, since it is documented as a separate field. In this case they did not match - exactly why we try not to specify twice.